<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="lineBefore">
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineNow">
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>116</width>
        <height>83</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnPRE">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnCE">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>CE</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnC">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnBAC">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnREC">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>1/x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnSQR">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>sqr</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnSQRT">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>sqrt</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnDIV">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn7">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn8">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btn9">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btnMUL">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btn4">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btn5">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="btn6">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="btnSUB">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btn1">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="btn2">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="btn3">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="btnADD">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="btnOPP">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>+/-</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="btn0">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="btnPOI">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="btnRES">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton:hover{
    background-color: rgba(152, 188, 255, 0.6);
}
.QPushButton{
	background-color: rgba(198, 218, 255, 0.6);
}</string>
         </property>
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
